addons/l10n_fr_invoice_addr/views/report_invoice.xml

```diff
@@ -0,0 +1,81 @@
+            <t t-set="partner" t-if="not o.l10n_fr_is_company_french()" t-value="o.partner_id"/>
+            <address t-field="partner.self" class="m-0"
+                     t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
+
+            <t t-if="o.l10n_fr_is_company_french() and partner.siret">
+                SIRET: <t t-esc="partner.siret"/>
+            </t>
+
+            <div class="mb-3"/>
+        </xpath>
+
+        <!--
+            INFORMATIONS BLOCK
+        -->
+
+        <!-- Shipping Address -->
+        <xpath expr="//div[@name='shipping_address_block']" position="attributes">
+            <attribute name="groups"/>
+            <attribute name="t-if">
+                o.l10n_fr_is_company_french() or o.env.user.has_group('sale.group_delivery_invoice_address')
+            </attribute>
+        </xpath>
+
+        <!-- Billing Address -->
+        <xpath expr="//div[@id='informations']" position="inside">
+            <t t-if="o.l10n_fr_is_company_french() and o.partner_id.parent_id">
+                <div class="col-auto col-3 mw-100 mb-2">
+                    <div class="font-weight-bold">Billing Address:</div>
+                    <p class="m-0">
+                        <address t-field="o.partner_id" class="m-0"
+                                 t-options='{"widget": "contact", "fields": ["address"], "no_marker": True}'/>
+                    </p>
+                </div>
+            </t>
+        </xpath>
+
+        <!-- Product types -->
+        <xpath expr="//div[@id='informations']" position="inside">
+            <t t-if="o.l10n_fr_is_company_french()">
+                <t t-set="tax_scopes" t-value="o.invoice_line_ids.mapped('tax_ids.tax_scope')"/>
+                <t t-set="has_service" t-value="'service' in tax_scopes"/>
+                <t t-set="has_consu" t-value="'consu' in tax_scopes"/>
+
+                <t t-if="has_service or has_consu">
+                    <div class="col-auto col-3 mw-100 mb-2">
+                        <div class="font-weight-bold">Operation Type:</div>
+                        <t t-if="has_service and has_consu">
+                            <p class="m-0">Mixed Operation</p>
+                        </t>
+                        <t t-elif="has_service and not has_consu">
+                            <p class="m-0">Service Delivery</p>
+                        </t>
+                        <t t-else="">
+                            <p class="m-0">Goods Delivery</p>
+                        </t>
+                    </div>
+                </t>
+            </t>
+        </xpath>
+
+        <!-- Payment according to debits mention -->
+        <xpath expr="//p[@name='payment_communication']" position="after">
+            <t t-if="o.l10n_fr_is_company_french()">
+                <t t-if="'on_invoice' in o.invoice_line_ids.mapped('tax_ids.tax_exigibility')">
+                    <p>Option to pay tax on debits</p>
```

Comment:
use of P ?

Reply:
It gets inserted right in the middle of other lines which are using `p` so it'd make sense to use it here too for consistency